homeassistant/components/knx/strings.json

```diff
@@ -0,0 +1,42 @@
+{
+  "config": {
+    "step": {
+      "type": {
+        "description": "Please enter the connection type we should use for your KNX connection. \n AUTOMATIC - The integration takes care of the connectivity to your KNX Bus by performing a gateway scan. \n TUNNELING - The integration will connect to your KNX bus via tunneling. \n ROUTING - The integration will connect to your KNX bus via routing.",
+        "data": {
+          "connection_type": "KNX Connection Type"
+        }
+      },
+      "tunnel": {
+        "description": "Please select a gateway from the list.",
+        "data": {
+          "gateway": "KNX Tunnel Connection"
+        }
+      },
+      "manual_tunnel": {
+        "description": "Please enter the connection information of your tunneling device.",
+        "data": {
+          "port": "[%key:common::config_flow::data::port%]",
+          "host": "[%key:common::config_flow::data::host%]",
+          "individual_address": "Individual address for the connection",
+          "route_back": "When True the KNXnet/IP Server shall use the IP address and the port number from the IP package received as the target IP address or port number for the response to the KNXnet/IP Client (for NAT/Docker)"
```

Comment:
No need to include all details of the hack. Just say that people might want to check it in case they have some issues and use a container installation (maybe even only show it then?)

Reply:
I copied that from the documentation, I'll change it a bit.

Reply:
I wouldn't mention Containers explicitly as the HA containers are supposed to run with `net=host` anyway - and this works fine without route_back. 
I'd just call it "route back" which is the name used in KNX specifications or "NAT mode" which, I think, is used in ETS. Or a combination of these 2.